<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="title">
       <string>原始数据筛选</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>开始日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="startDateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>结束日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="endDateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2014</year>
               <month>12</month>
               <day>30</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="daySumRadio">
               <property name="text">
                <string>日和</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="weekSumRadio">
               <property name="text">
                <string>周和</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="monthSumRadio">
               <property name="text">
                <string>月和</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="typeCombo"/>
           </item>
           <item>
            <widget class="QComboBox" name="numCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="queryBtn">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="title">
       <string>计算筛选</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>比较日期：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="cmpDateEdit1">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>——</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="cmpDateEdit2">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>比较方法</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmpMethCombo"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>比较指标：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmpTypeCombo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>结果筛选：</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>&gt;=</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>&lt;=</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="calculateBtn">
           <property name="text">
            <string>计算</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>原始数据：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="FreezeTableWidget" name="tableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="tb1_headBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1_upBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>上页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tb1_curPageLabel">
             <property name="text">
              <string>0/0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1_downBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>下页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1_lastBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>尾页</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>n日数据</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="FreezeTableWidget" name="tableView_2"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="tb2_headBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2_upBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>上页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tb2_curPageLabel">
             <property name="text">
              <string>0/0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2_downBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>下页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2_lastBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>尾页</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>计算结果</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="FreezeTableWidget" name="tableView_3"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_showGroupView"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QDockWidget" name="groupView">
   <property name="windowTitle">
    <string>股票筛选</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>分组1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>分组2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>设置列</string>
   </property>
  </action>
  <action name="action_addGroup">
   <property name="text">
    <string>添加分组</string>
   </property>
  </action>
  <action name="action_editGroup">
   <property name="text">
    <string>编辑分组</string>
   </property>
  </action>
  <action name="action_showGroupView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>股票筛选</string>
   </property>
  </action>
  <action name="action_deleteGroup">
   <property name="text">
    <string>删除分组</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FreezeTableWidget</class>
   <extends>QTableView</extends>
   <header>freezetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
